test/tests.spec.ts

```diff
@@ -701,7 +701,7 @@ describe("node-saml /", function () {
         const base64xml = Buffer.from(xml).toString("base64");
         const container = { SAMLResponse: base64xml };
         const samlObj = new SAML({
-          cert: "-----BEGIN CERTIFICATE-----" + TEST_CERT + "-----END CERTIFICATE-----",
+          cert: `-----BEGIN CERTIFICATE-----\n${TEST_CERT}\n-----END CERTIFICATE-----`,
```

Comment:
This technically changes the test. Will the code still work if there isn't a newline after the opening segment or before the closing segment? I know that is non-standard, but, since it is supported that way in a test, we should probably continue supporting it until a semver-major.